moveit_core/CMakeLists.txt

```diff
@@ -102,59 +99,61 @@ set(THIS_PACKAGE_LIBRARIES
     moveit_constraint_samplers
     moveit_planning_request_adapter
     moveit_profiler
-    # TODO: Port python bindings
-    # moveit_python_tools
+    # TODO: Port python bindings moveit_python_tools
     moveit_trajectory_processing
     moveit_distance_field
     moveit_collision_distance_field
     moveit_kinematics_metrics
     moveit_dynamics_solver
     moveit_utils
-    moveit_test_utils
-)
+    moveit_test_utils)
 
 set(THIS_PACKAGE_INCLUDE_DEPENDS
-  angles
-  eigen_stl_containers
-  geometric_shapes
-  geometry_msgs
-  kdl_parser
-  moveit_msgs
-  octomap_msgs
-  random_numbers
-  sensor_msgs
-  shape_msgs
-  srdfdom
-  std_msgs
-  tf2_eigen
-  tf2_geometry_msgs
-  trajectory_msgs
-  visualization_msgs
-  Eigen3
-  eigen3_cmake_module
-  OCTOMAP
-  Bullet
-)
-
-pluginlib_export_plugin_description_file(moveit_core collision_detector_fcl_description.xml)
-pluginlib_export_plugin_description_file(moveit_core collision_detector_bullet_description.xml)
-
-include_directories(SYSTEM ${EIGEN3_INCLUDE_DIRS}
-  ${LIBFCL_INCLUDE_DIRS}
-)
-
-include_directories(${THIS_PACKAGE_INCLUDE_DIRS}
-                    ${VERSION_FILE_PATH})
+    angles
+    eigen_stl_containers
+    geometric_shapes
+    geometry_msgs
+    kdl_parser
+    moveit_msgs
+    octomap_msgs
+    random_numbers
+    sensor_msgs
+    shape_msgs
+    srdfdom
+    std_msgs
+    tf2_eigen
+    tf2_geometry_msgs
+    trajectory_msgs
+    visualization_msgs
+    Eigen3
+    eigen3_cmake_module
+    OCTOMAP
+    Bullet)
```

Comment:
Personally I prefer the `)` in a separate line, but I see how that is bike-shedding